src/extension/autoconverter/Detect.re

```diff
@@ -0,0 +1,58 @@
+open Core;
+open Core.Dom;
+open Common;
+
+let hasClassName className =>
+  List.length (getElementsByClassName None className) > 0;
+
+let ocamlishRels = [|
+  "Start",
+  "previous",
+  "next",
+  "Up",
+  "Appendix",
+  "Section",
+  "Subsection"
+|];
+
+let hasOcamlRels => {
+  let rels =
+    getElementsByTagName None "link"
+      |> List.map (fun link => ocamlishRels |> arrayContains (Element.getAttribute link "rel"))
+      |> List.filter (fun b => b)
+      |> List.length;
+
+    rels >= 3
+};
+
+let hasCommonClassNames =>
+  [
+    "keyword",
+    "type",
+    "deprecated",
+    "mod",
+    "modt",
+    "typ",
+    "spec",
+    "def",
+    "ext",
+    "exn",
+    "cls",
+    "include",
+    "cons",
+    "paramstable",
+    "sig_block"
+  ]
+  |> List.map hasClassName
+  |> List.for_all (fun b => b);
```

Comment:
Hey I think I actually missed this in the last PR, but how is this working? In the JS version this checks to make sure at least 3 class names existed - https://github.com/reasonml/reason-tools/blob/c568abf134bae976080c4a5870f2c66eedcf92b7/src/AutoConversionPrompt.js#L60

This implementation doesn't look like it does that. Am I missing something?